src/views/village/content/ArmyCard.vue

```diff
@@ -0,0 +1,62 @@
+<template>
+    <el-row :gutter="24">
+            <el-col :span="4">
+                <el-image :src="require('@/assets/image/units/pic.png')"
+                          :fit="'scale-down'"
+                          alt="Wood"
+                          class="army-image"
+                ></el-image>
+                {{infantry.pikes}}
+            </el-col>
+
+            <el-col :span="5">
+                <el-image :src="require('@/assets/image/units/sword.png')"
+                          :fit="'scale-down'"
+                          alt="Wood"
+                          class="army-image"
+                ></el-image>
+                {{infantry.swords}}
+            </el-col>
+
+            <el-col :span="5">
+                <el-image :src="require('@/assets/image/units/axe.png')"
+                          :fit="'scale-down'"
+                          alt="Wood"
+                          class="army-image"
+                ></el-image>
+                {{infantry.axes}}
+            </el-col>
+
+            <el-col :span="5">
+                <el-image :src="require('@/assets/image/units/light.png')"
+                          :fit="'scale-down'"
+                          alt="Wood"
+                          class="army-image"
+                ></el-image>
+                {{cavalry.light}}
+            </el-col>
+
+            <el-col :span="5">
+                <el-image :src="require('@/assets/image/units/heavy.png')"
+                          :fit="'scale-down'"
+                          alt="Wood"
+                          class="army-image"
+                ></el-image>
+                {{cavalry.heavy}}
+            </el-col>
```

Comment:
You can make an array of {UnitlType, ImagePath and ColspanValue } objects, and then use v-for to loop over them in order to remove redundancy. Also, you have the same alt value everywhere